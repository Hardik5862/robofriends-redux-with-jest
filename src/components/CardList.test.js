import React from "react";
import { shallow } from "enzyme";
import CardList from "./CardList";

it("expect to render cardlist component", () => {
  const mockRobots = [
    {
      id: 1,
      name: "Sabrina",
      email: "sabrina@gmail.com",
    },
  ];

  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});
