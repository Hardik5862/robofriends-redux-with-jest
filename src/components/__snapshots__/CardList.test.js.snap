// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render cardlist component 1`] = `
<div>
  <Card
    email="sabrina@gmail.com"
    id={1}
    key="0"
    name="Sabrina"
  />
</div>
`;
